pages/project/content/index.js

```diff
@@ -63,5 +63,14 @@ module.exports = {
     transferred: 'Transferred',
     expired: 'Expired',
     revoked: 'Revoked'
+  },
+  collaborators: {
+    title: 'Colleagues granted access',
+    fields: {
+      name: 'Name',
+      email: 'Email',
+      action: ''
```

Comment:
?

Reply:
yeah, in the content doc we have removed the heading for actions